designs/virtualization/4.x/snapshots/snapshots.md

```diff
@@ -0,0 +1,85 @@
+---
+parent: Virtualization
+version: 4.x
+---
+
+# VM snapshots
+
+This item will cover updates to vm-snapshots
+- Creating a VM snapshot
+- Snapshots on the VM details page
+- Virtualization snapshot list
+- Restoring a VM from snapshot
+
+## Creating a snapshot
+
+![Virtualization - VMs list](img/0-0.jpg)
+From the virtual machines menu item, the user selects “Create Snapshot” from a VM item’s kebab dropdown.
+The same action can be taken from a VM details Action dropdown.
+
+![Virtualization - turn off VM notification](img/0-1.jpg)
+In case the VM is running, the user will be asked to turn it off to make a snapshot
+
+![Virtualization - Create VM snapshot](img/1-0.jpg)
+The user enters the Create snapshot screen.
+This screen includes:
+- Snapshot name field
+- Save memory content (future implementation)
```

Comment:
can we separate out these "Future implementation" items? It would be good to have it assigned to 4.x so we don't forget about it.

Reply:
ah I just realized these are all under 4.x so maybe we don't need to say these are future implementations? OR if we don't think it will get in whenever its implemented we should provide designs that cover what will go in first. WDYT?

Reply:
I think I better drop it. 
As for the Included disks @aglitke mentioned it is not implemented yet. but the disks are included in the snapshot.
I think it is ok to keep that dropdown and drop the "Future version" from it, but I'm waiting to hear which version will this dropdown go into.

Reply:
@matthewcarleton So the snapshots should be for 4.6 but it may be ready for 4.7.
So I am placing it ina 4.7 for now.
I also removed the "Included disks" drop-down and the "Include memory" checkbox and place it as a new MD in 4.x.
I'm using this PR for both MDs. Let me know if that's ok.

Reply:
perfect! so @jelkosz was saying today that we will see *something* for 4.6. Do we know what that is? Should we just include that for the 4.6 and then just put everything else in 4.x for now?